src/App/App.css

```diff
@@ -17,4 +17,23 @@
 
 .App-link {
   color: #61dafb;
+}
+
+.page-enter {
+  opacity: 0;
```

Comment:
Fixing that "ghost page" on the flip helps a lot with the user experience, and helps it feel more polished. Great job figuring this out!